### PR TITLE
Design: Go-Kit migration

### LINKED ISSUE #76: Design: Go-Kit migration
Adapt [Go-Kit](https://gokit.io) on the current implementation.

### PR DESCRIPTION
**Pull request by 777777miSSU7777777**
Adapt Go-Kit on the current implementation.
Closes #76.

### CODE REVIEW

**Comment by 777777miSSU7777777** [comment]
> # [Codecov](https://codecov.io/gh/777777miSSU7777777/github-aggregator/pull/80?src=pr&el=h1) Report
> > Merging [#80](https://codecov.io/gh/777777miSSU7777777/github-aggregator/pull/80?src=pr&el=desc) into [master](https://codecov.io/gh/777777miSSU7777777/github-aggregator/commit/13c8818235bf134b417783541fcc400572bf6d94?src=pr&el=desc) will **not change** coverage.
> > The diff coverage is `n/a`.
> 
> ```diff
> @@          Coverage Diff           @@
> ##           master     #80   +/-   ##
> ======================================
>   Coverage    73.7%   73.7%           
> ======================================
>   Files          16      16           
>   Lines         213     213           
> ======================================
>   Hits          157     157           
>   Misses         54      54           
>   Partials        2       2
> ```

**Comment by 777777miSSU7777777** [comment]
@amaskalenka 

> Where is Go-Kit?

Will be added with endpoints and transport.

File: Makefile [hunk 1]
```diff
@@ -5,15 +5,14 @@ check:
 
 	go list ./... | grep -v /vendor/ | xargs -n 1 golint
 
-	gosec ./...
+	gosec -exclude=G104 ./...
```
Comment (amaskalenka) [inline Makefile]: Please add comment why you are excluding this rule.

File: internal/api/api.go [hunk 2]
```diff
@@ -3,12 +3,20 @@ package api
...
 	"net/http"
+
+	log "github.com/sirupsen/logrus"
 )
 
 var httpGet = http.Get
 
 var client *http.Client
 
+var logger *log.Logger
```
Comment (amaskalenka) [inline internal/api/api.go]: Are you going to have logger in each and every package? Please don't use global variables, pass logger as a parameter/add to context/include in the types as a field

**Pull request merged**